src/handlers/decision.rs

```diff
@@ -106,6 +118,21 @@ pub(super) async fn handle_command(
                     )
                     .await?;
 
+                    let metadata = serde_json::value::to_value(DecisionProcessActionMetadata {
+                        message: "some message".to_string(),
+                        get_url: format!("{}/issues/{}", issue.repository().url(), issue.number),
+                        status: Merge,
+                    })
+                    .unwrap();
```

Comment:
NIT: what do you think about naming `get_url` something like `get_issue_url`? to specify what the `get` is for - I know the formatting is very self-explanatory, so no big deal.